static/js/loader.js

```diff
@@ -0,0 +1,15 @@
+var loader = (function () {
+
+var exports = {};
+
+exports.show_loader = function (btn_id) {
+   console.log("I am in");
+}
+
+return exports;
+
+}());
+
+if (typeof module !== 'undefined') {
+    module.exports = loader;
+}
```

Comment:
There's no need for a new file; just add your changes in the `signup.js` file I mentioned.